packages/babel-parser/test/fixtures/experimental/import-assertions/valid-empty-assertion/input.js

```diff
@@ -0,0 +1 @@
+import "foo" assert {};
```

Comment:
I meant specifically, `import { x } from "foo" assert {}` shouldn't be an error because even if we have assertions they don't contain `type: "json"`.